e2e/e2e-prow.sh

```diff
@@ -53,13 +53,17 @@ properties=(
   -Dspark.kubernetes.test.imageRepo=$IMAGE_REPO \
   -Dspark.kubernetes.test.sparkTgz="$SPARK_TGZ" \
   -Dspark.kubernetes.test.deployMode=cloud \
-  -Dspark.kubernetes.test.namespace=default \
-  -Dspark.kubernetes.test.serviceAccountName=default
+  -Dspark.kubernetes.test.namespace=spark \
+  -Dspark.kubernetes.test.serviceAccountName=spark-sa
 )
 
+# Run kubectl commands and create appropriate roles
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user pr-kubekins@kubernetes-jenkins-pull.iam.gserviceaccount.com
```

Comment:
Do you need to remote the ClusterRole `spark-role` in `spark-rbac.yaml`?

Reply:
I don't think so - this one is to grant the GCP service account the ability to create the rbac.yaml's spark-role.

Reply:
I see.